core/templates/dev/head/components/forms/HtmlSelectDirective.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the select supporting html
+ * @fileoverview Directive for the select supporting html.
```

Comment:
Directive for the selection dropdown with HTML content.

Reply:
done